appsync-eventbridge-datasource/README.md

```diff
@@ -0,0 +1,55 @@
+# Appsync to EventBridge Datasource 
+
+This pattern demonstrates sending events directly to Event Bus by integrating Appsync with Eventbridge Datasource.
```

Comment:
there is an image in the project that is not used in the readme. Maybe you can add it

Reply:
Added architecture diagram to readme